python/metatensor-torch/metatensor/torch/atomistic/systems_to_torch.py

```diff
@@ -0,0 +1,117 @@
+from typing import List, Optional, Union
+
+import numpy as np
+import torch
+
+from . import System
+
+
+try:
+    import ase
+
+    HAS_ASE = True
+except ImportError:
+    HAS_ASE = False
+
+
+class IntoSystem:
+    """A type that can be converted into a ``metatensor.torch.atomistic.System``.
+
+    This is an abstract class that is used to indicate a class whose objects
+    can be converted into a ``metatensor.torch.atomistic.System``. For the moment,
```

Comment:
```suggestion
    can be converted into a :py:class:`metatensor.torch.atomistic.System`. For the moment,
```

Might also work like this

```suggestion
    can be converted into a :py:class:`System`. For the moment,
```